docs/api.rst

```diff
@@ -5806,7 +5806,7 @@ Export PnL report debug data
                     "uniswapv2",
                     "uniswapv3",
                 ],
-                "historical_price_oracles": ["manual", "cryptocompare", "coingecko"],
+                "historical_price_oracles": ["cryptocompare", "coingecko"],
```

Comment:
add some more of the new ones in the example. Such as defillama, alchemy, uniswapv2 and uniswapv3